sys/net/gnrc/application_layer/zep/gnrc_zep.c

```diff
@@ -573,7 +573,7 @@ void *_event_loop(void *args)
         switch (msg.type) {
             case GNRC_NETAPI_MSG_TYPE_RCV:
                 DEBUG("zep: GNRC_NETAPI_MSG_TYPE_RCV\n");
-                ringbuffer_add(&_rx_buf, (char *)(&msg.content.ptr),
+                ringbuffer_add(&_rx_buf, (void*)&msg.content.ptr,
```

Comment:
isn't the same true for `void**` as for `void*`? Do you actually need to cast if `void*` would not need it?